src/main/java/core/basesyntax/interfaces/FileWriterTool.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+
+public interface FileWriterTool {
+    void writeReport(String data, String reportPath);
```

Comment:
You are writing to file, it can be not only report but other files, so make method name more generic.